CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable, unreleased changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Breaking Changes
```

Comment:
Deprecating a field is not breaking, removing a field is. When a field becomes deprecated, it's a notice for API consumers that they need to migrate to another field or stop using it, but their implementations won't break at this point.